docs/userGuide/syntax/boxes.mbdf

```diff
@@ -223,7 +223,7 @@ dismissible | `Boolean` | `false` | Adds a button to close the box to the top ri
 icon{{slot_info_trigger}} | `String` | `null` | Inline MarkDown text of the icon displayed on the left.
 icon-size | `String` | `null` | Resizes the icon. Supports integer-scaling of the icon dimensions e.g. `2x`, `3x`, `4x`, etc.
 icon-color | `String` | `null` | Color of the icon.
-header{{slot_info_trigger}} <hr style="margin-top:0.2rem; margin-bottom:0" /> <small>heading <br> (deprecated)</small> | `String` | `null` | Markdown text of the box header.
+header{{slot_info_trigger}} <hr style="margin-top:0.2rem; margin-bottom:0" /> | `String` | `null` | Markdown text of the box header.
```

Comment:
```suggestion
header{{slot_info_trigger}} | `String` | `null` | Markdown text of the box header.
```
The `<hr>` tag can be removed as the line is no longer needed. Same for the rest :)

![image](https://user-images.githubusercontent.com/47470981/149937881-c5e130b8-37c5-46f6-b26f-c1ecb9750da8.png)

Reply:
Thanks for pointing this out!